.github/workflows/dockerhub-master.yml

```diff
@@ -33,7 +33,7 @@ jobs:
         run: doctl kubernetes cluster kubeconfig save ${{ secrets.K8S_CLUSTER }}
 
       - name: Re-apply manifests
-        run: kubectl apply -n development -k $GITHUB_WORKSPACE/k8s/coordinator/development --prune -l "app.kubernetes.io/part-of=xaynet"
```

Comment:
Turns out that `--prune` is still in alpha and this doesn't _quite_ do what I initially though, so reverting the change.